fix(curriculum): make the description clearer for discordian-date (#47739)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md

---

# --description--

> The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. A week consists of seven days, and there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February every four years (except for three centurial years for every 400 years).

Reviewer: ```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. A week consists of seven days, and there are 52 weeks and 1 day per year. The year consists of 365 days with leap day is added to February every leap year. A leap year occurs every four years, except for the first three centurial years of every 400 years.
```

> The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February every four years, which is called a leap year (except for three centurial years for every 400 years).

Reviewer: ```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. A week consists of seven days, and there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February every four years (except for three centurial years for every 400 years).
```

> The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February every four years, which is called a leap year (except for three centurial years for every 400 years).

Reviewer 2: ```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with a leap day added to February in leap years (these are every 4 years, in years divisible by 4, but if the year ends with 00 it is a leap year only if it's divisible by 400).
```

Reviewer: Can we rework to avoid the parenthetical statement? I.e. pull that text out into sentences outside of the ()

Reviewer 2: ?
```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. The week here consists of seven days, whereas there are 52 weeks and 1 day per year. The year consists of 365 days, with an extra day added to February in leap years.

Leap years happens every 4 years, in years divisible by 4, but if the year ends with 00 it is a leap year only if it's divisible by 400
```

> The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. A week consists of seven days, and there are 52 weeks and 1 day per year. The year consists of 365 days with leap day is added to February every leap year. A leap year occurs every four years, except for the first three centurial years of every 400 years.

Reviewer 3: ```suggestion
The Gregorian calendar is a solar calendar with 12 months of 28-31 days each. A week consists of seven days, and there are 52 weeks and 1 day per year. The year consists of 365 days with a leap day added to February every leap year. A leap year occurs every four years, except for the first three centurial years of every 400 years.
```

> The <a href="http://www.rosettacode.org/wiki/Discordian_date" target="_blank" rel="noopener noreferrer nofollow">Discordian calendar</a> is aligned with the Gregorian calendar, and begins on January 1. The week consists of five days, and there are 73 weeks per year. The year also consists of 365 days, with an extra day inserted between Chaos 59 and Chaos 60 every four years (called a leap year).

Reviewer: With above we can simplify to
```suggestion
The <a href="http://www.rosettacode.org/wiki/Discordian_date" target="_blank" rel="noopener noreferrer nofollow">Discordian calendar</a> is aligned with the Gregorian calendar, and begins on January 1. The week consists of five days, and there are 73 weeks per year. The year also consists of 365 days, with an extra day inserted between Chaos 59 and Chaos 60 every leap year.
```

> The <a href="http://www.rosettacode.org/wiki/Discordian_date" target="_blank" rel="noopener noreferrer nofollow">Discordian calendar</a> is aligned with the Gregorian calendar, and begins on January 1. The week consists of five days, whereas there are 73 weeks per year. The year also consists of 365 days, with an extra day inserted between Chaos 59 and Chaos 60 every four years (called a leap year).

Reviewer: ```suggestion
The <a href="http://www.rosettacode.org/wiki/Discordian_date" target="_blank" rel="noopener noreferrer nofollow">Discordian calendar</a> is aligned with the Gregorian calendar, and begins on January 1. The week consists of five days, and there are 73 weeks per year. The year also consists of 365 days, with an extra day inserted between Chaos 59 and Chaos 60 every four years (called a leap year).
```

> The months, weekdays, and Apostle events and Holidays in the Discordian calendar are given by:

Reviewer: ```suggestion
The months, weekdays, Apostle events, and Holidays in the Discordian calendar are given by:
```
Duh, this is clearer, my bad

> The months, weekdays, and events (Apostle and Holidays) in the Discordian calendar are given by:

Reviewer: ```suggestion
The months, weekdays, and Apostle events and Holidays in the Discordian calendar are given by:
```
